Sources/StreamChatUI/ChatMessageList/Reactions/ChatMessageReactionAuthorsVC/ChatMessageReactionAuthorsVC_Tests.swift

```diff
@@ -77,10 +77,25 @@ final class ChatMessageReactionAuthorsVC_Tests: XCTestCase {
             cid: .unique,
             text: "Some text",
             author: .mock(id: .unique),
-            reactionCounts: ["fake": 1]
+            reactionCounts: ["like": 1]
         )
         messageControllerMock.reactions = [
-            .mock(author: .mock(id: .unique, imageURL: TestImages.vader.url))
+            .mock(type: "like", author: .mock(id: .unique, imageURL: TestImages.vader.url))
+        ]
+        AssertSnapshot(vc, variants: [.defaultLight])
+    }
+
+    func test_defaultAppearance_shouldNotRenderUnavailableReactions() {
+        messageControllerMock.message_mock = .mock(
```

Comment:
nit: would be nice to explicitly assign `Appearance.images.availableReactions` so it's clear which reactions are supposed to be rendered and which are not

Reply:
I thought about this, but it is a bit too much code to set this up, so I didn't do it. Unfortunately, we will need to rely on the real available reactions.